### DISCUSSION TITLE
Relative Leaderboards

### DISCUSSION THREAD

**Discussion by RealistikDash** [ppy/osu#21779] [Ideas] [answered]
## The Idea
I propose the idea of "relative leaderboards" to be added alongside the existing leaderboards. This leaderboard type will show scores around the user's existing top score (eg 25 scores above and 24 below).

## The Reasoning
Currently, osu only displays the top 50 scores for all its leaderboards. This means that if you are not a top player, it is almost impossible to see how your score compares to others aside from a very vague placement number. The implementation of relative leaderboards is likely to incentivise more competition within leaderboard rankings, as players are provided with direct feedback on their performance relative to others around them.

## Potential Issues
Relative leaderboards are likely to computationally expensive for the servers. The current leaderboard implementation means that every player is accessing the same set of leaderboards, meaning it is relatively easy to cache them. Relative leaderboards however are personalised for every player, making them much more computationally expensive with osu's large database.

However, a potential solution for this would be to make relative leaderboards a supporter feature, similarly to existing leaderboards.

On popular maps, scores in certain rank ranges can be *very* similar.

This can potentially be addressed by showing eg every 10th score.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/17673
